app/assets/stylesheets/error-page.scss

```diff
@@ -1,2 +1,8 @@
 $govuk-compatibility-govuktemplate: true;
-@import "govuk_publishing_components/all_components";
+$govuk-use-legacy-palette: false;
+
+@import "govuk_publishing_components/govuk_frontend_support";
+@import "govuk_publishing_components/component_support";
```

Comment:
Presumably component_support is needed for the button sass? Otherwise this line could be removed.

Reply:
Yeah, without it I get the error: `Undefined mixin 'responsive-bottom-margin'.`

Reply:
Fair enough then.